HLTrigger/Egamma/plugins/HLTEgammaCombMassFilter.cc

```diff
@@ -45,6 +47,35 @@ bool HLTEgammaCombMassFilter::hltFilter(edm::Event& iEvent,
                                         trigger::TriggerFilterObjectWithRefs& filterproduct) const {
   //right, issue 1, we dont know if this is a TriggerElectron, TriggerPhoton, TriggerCluster (should never be a TriggerCluster btw as that implies the 4-vectors are not stored in AOD)
```

Comment:
Does this comment still apply after the changes proposed here?

Reply:
I think this problem should already be solved after we applied the proposed change, but one can verify if this is really solved or not.

Reply:
If it does, then please remove the comment.

Reply:
Sure, I have removed the comment. Thanks for the reminder!